accessibility/demo/src/main/res/values/ids.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <item name="tweet_action_open"
```

Comment:
it's enforced by something, not sure if it was on purpose in an update of the novoda code styles or if I've broken something.